draft-ietf-snac-simple.xml

```diff
@@ -206,6 +206,13 @@
 	<dt>RA Beacon:</dt>
 	<dd>A Router Advertisement (RA) that is multicast on a link so that hosts can see that the router is still present. This is in
 	  contrast to a unicast RA sent in response to the router solicit.</dd>
+	<dt>ULA Site Prefix:</dt>
+	<dd>A Unique Local Address /48 prefix <xref target="RFC4193"/> randomly generated by each stub router for use in providing
+	  ULA Link Prefixes to the stub network and the adjacent infrastructure link, as needed.</dd>
+	<dt>ULA Link Prefix:</dt>
+	<dd>A Unique Local Address /64 prefix allocated from the ULA Site Prefix that stub routers advertise to provide
```

Comment:
this is now a bit hard to parse. How about:
allocated from the ULA site prefix. Stub routers can use ULA Link prefixes to provide addressability on the stub network and/or adjacent infrastructure link as needed. If a BR is doing NAT64, the NAT64 prefix is also a ULA Link Prefix. A total of....

Reply:
Updated.